lib/phone_number_formatter.rb

```diff
@@ -1,6 +1,17 @@
 # This method takes a string like `"4122226644"` and
 # returns a properly formatted phone number.
+def select_only_integers(phone_number_str)
+	#remove all the non integers from the string
+	phone_number = {}
+	phone_number = phone_number_str.gsub(/\D/, '')	
+	return phone_number
+end
+
 
 def format_phone_number(phone_number_str)
-  
+	#assign the method from above
+	phone_number = select_only_integers
```

Comment:
Your `select_only_integers` method takes one argument. While it seems like it would automatically figure out what `phone_number_str` is, it actually doesn't. Each method is essentially walled off from others. So if you are going to use `select_only_integers` in the `format_phone_number` method, you will need to pass the phone number string through to it.

So try `phone_number = select_only_integers(phone_number_str)`